shared/src/main/scala-2.11/scala/util/parsing/input/PagedSeq.scala

```diff
@@ -0,0 +1 @@
+../../../../../scala-2.12/scala/util/parsing/input/PagedSeq.scala
```

Comment:
I moved `PagedSeq.scala` to the `src/main/scala-2.12` source directory and symlinked it from here (`scala-2.11` source directory)

Reply:
This probably doesn't work on Windows. Alternatively we can do something like https://github.com/sbt/sbt/issues/2310

Reply:
Done!